recipes/apicurio-keycloak-extensions/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apicurio-keycloak-extensions" %}
+{% set version = "9.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/Apicurio/{{ name }}/archive/{{ version }}.Final.tar.gz
+  sha256: a95686ada376b97d205a6b9b80403b6107891d01878260f155633e5e04a9696a
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - set -x
+    - mvn package
+    - mkdir -p $PREFIX/jars                                                                             # [unix]
+    - mkdir "%PREFIX%\jars"                                                                             # [win]
+    - cp social/target/apicurio-keycloak-extensions-social-{{ version }}.Final.jar $PREFIX/jars         # [unix]
+    - copy "social\target\apicurio-keycloak-extensions-social-{{ version }}.Final.jar" "%PREFIX%\jars"  # [win]
```

Comment:
Please just make a build.sh file. As noarch generic it doesn't need to build on win

Reply:
Can do

Reply:
Do I need to add a skip for win?

Reply:
nah, you don't need to skip win